### ISSUE TITLE
Don't disable firewall

### ISSUE THREAD

**Issue by ofrzeta** [kafecho/ansible-couchdb2#2]
I think it's better not to disable the firewall. The package is just about building CouchDB. Let the user handle the firewall instead of just disabling it and open a potential security risk. Or at least place big disclaimer in the README.

**Comment by kafecho** [comment]
Good catch. I will add some notes to the disclaimer and add a task to tweak iptables. I see there is a new module for that in Ansible 2.0.
